### PR TITLE
Replace `chalk` with `picocolors`

### PR DESCRIPTION
**Pull request by trivikr**
Fixes: https://github.com/changesets/changesets/issues/1414

### CODE REVIEW

File: packages/cli/src/commands/add/messages.ts [hunk 1]
```diff
@@ -21,18 +21,19 @@ export default function printConfirmationMessage(
...
-    log(`${chalk.bold.green("minor")}:  ${minorReleases.join(", ")}`);
+    log(`${pc.bold(pc.green("minor"))}:  ${minorReleases.join(", ")}`);
   if (patchReleases.length > 0)
-    log(`${chalk.bold.green("patch")}:  ${patchReleases.join(", ")}`);
+    log(`${pc.bold(pc.green("patch"))}:  ${patchReleases.join(", ")}`);
 
   log("");
 
   if (repoHasMultiplePackages) {
+    // ToDo: Change to pc.redBright when https://github.com/alexeyraspopov/picocolors/pull/55 is published.
```
Comment (Andarist) [inline packages/cli/src/commands/add/messages.ts:33]: I wouldn't block this PR on this and I would be OK with just switching to `pc.red` here. But given that it seems that the referenced PR might get published soon-ish... it feels like it's best to wait for it.

It that won't happen in like a week or two then we can drop this TODO and land this PR as-is

Comment (trivikr) [reply packages/cli/src/commands/add/messages.ts:33]: Yeah. Let's wait for new version of piccocolors to be published for a week or two.

Comment (benmccann) [reply packages/cli/src/commands/add/messages.ts:33]: @trivikr the new release of picocolors is now out!

Comment (trivikr) [reply packages/cli/src/commands/add/messages.ts]: PR is updated to use `pc.redBright` in [`dcc7754` (#1417)](https://github.com/changesets/changesets/pull/1417/commits/dcc77542a324d290cb5a950553fe8f895b75a16c)

**Comment by trivikr** [comment]
@Andarist The PR is ready for review post picocolors@1.1.0 update.

**Pull request merged**